source/overview/auth.rst

```diff
@@ -1,69 +1,78 @@
-==================================
-Corporate Directory Integration 
-==================================
+============================
+Authentication and Security
+============================
 
-Mattermost Enterprise Edition offers advanced security and authentication options for integrating with corporate directories. This guide outlines the options available. 
+Mattermost has two levels of authentication and security at present.
+Future versions of Mattermost will extend these two options,
+but for now, the two main classes of authentication and security are the Mattermost Team Edition and the Mattermost Enterprise Editions.
+The Mattermost Enterprise Editions have the distinct advantage of corporate directory integration. 
 
 Basic authentication 
-----------------------------------
+--------------------
 
-By default Mattermost offers authentication via email and password, which offers basic features including: 
+The Mattermost Team Edition uses email and password for authentication.
+Email and password may be used with any version of Mattermost.
+Users may reset a password by email,
+sign in to the web application, 
+desktop application,
+and mobile application with the same email and password,
+and email can be substituted for username along with the same password.
 
-- Account sign-up using email and password
-- Password reset via email
-- Ability to use same credentials to sign-in to web, desktop app and mobile app experiences 
-- Ability to use username and password in place of email and password
+Core Security
+-------------
 
-Security features for authentication 
--------------------------------------------
+The core security features straddle the Mattermost Team Edition and the Mattermost Enterprise Editions.
+In all versions of Mattermost users have the ability to:
 
-A core set of features is available with all authentication options to help increase security: 
+- `Set session length <https://docs.mattermost.com/administration/config-settings.html#id33>`_, allowing administrators to define how long a user can use Mattermost before needing to re-enter credentials. 
```

Comment:
For readability perhaps: "....how long a user can be logged in to Mattermost..."?

Reply:
Yes, simple and to the point, however, it does not imply the consequence, which is that you will be kicked off your session. While it might be tacitly understood, it might be better to just state is clearly and unequivocally, "re-enter credentials."